src/components/inline/InputText/InputText.tsx

```diff
@@ -0,0 +1,55 @@
+import * as PropTypes from "prop-types";
```

Comment:
It looks like it's non-trivial to drop the `x` on `.tsx`, since the TypeScript compiler relies on it in order to decide whether to use TSX parsing mode or normal TS parsing mode. Apparently the angle brackets subtly conflict with the generic type parameters (e.g. `MouseEvent<HTMLInputElement>`), so it makes the parsing harder when you don't know which syntax type is used in the file.

I'm fine with standardizing on .tsx for this whole codebase, despite Gatsby's convention of trying to omit the `x`.